docs/reference_guide.md

```diff
@@ -2681,6 +2682,18 @@ contains the memcopy'd value so the value at `print()` invocation time will be
 printed.  However for maps, only the handle to the map is queued up, so the
 printed map may be different than the map at `print()` invocation.
 
+# 24. `strftime()`: Formatted timestamp
+
+Syntax:
+- `strftime(const char *format, int nsecs)`
+
+This returns a formatted timestamp that is printable with `printf`. The format string should be supported by `strftime(3)`. Use format specifier "%s" when printing the return value. Note that `strftime` does not actually return a string.
```

Comment:
The "should" makes it sound optional. "must" is more accurate. See IETF RFC 2119: http://www.faqs.org/rfcs/rfc2119.html
```suggestion
This returns a formatted timestamp that is printable with `printf`. The format string must be supported by `strftime(3)`. Use format specifier "%s" when printing the return value. Note that `strftime` does not actually return a string.
```